api/api/views/oauth2_views.py

```diff
@@ -182,20 +193,15 @@ def get(self, request, format=None):
 
         # TODO: Replace 403 responses with DRF `authentication_classes`.
```

Comment:
Is this still accurate now that we're using `PermissionDenied`?

Reply:
@dhruvkb, do you think it's still accurate, and could you explain what does "replacing with `authentication_classes`" mean? Asking you because it looks like you've added it in https://github.com/WordPress/openverse/pull/1876

Reply:
This is being addressed separately in #3663.

Reply:
It's not part of that PR you've linked, Dhruv, to be clear. That PR handles a different thing entirely than permissions, it handles validation of credentials, not permissions. A small distinction, but to be clear, that does not have anything to do with _permissions_. 

This comment is still valid, because the manual `PermissionDenied` raising can be implemented in DRF Permissions class, and should be, to follow DRF conventions and separate the concerns.